app/assets/javascripts/components/form/form_field.jsx

```diff
@@ -11,6 +13,16 @@ class FormField extends Component {
     return {};
   }
 
+  handleChange(event) {
+    var node = React.findDOMNode(this.refs.container);
+    this.props.updateValue(node, node.value);
+  }
+
+  componentDidMount() {
+    var node = React.findDOMNode(this.refs.container);
```

Comment:
Let's use `ReactDOM.findDOMNode` instead of `React.findDOMNode` since the latter is deprecated!